### PR TITLE
added readme file

### PR DESCRIPTION
**Pull request by nicolasbelfis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+a page using the template you modified  
+The hubspot code is visible in Marketing -> files and templates -> Design tool  
+
+
+### Staging
+
+The next step is to push changes to staging account to allow sales and marketing people double check the impact made by your changes
+
+1. `git push origin <your-branch>`
+2. create a merge request to the `stage branch`
```
Comment (ewansheldon) [inline README.md:62]: wouldn't this be less confusing if this was called `staging`

also i think only the `branchname` should be in code block like:

2. create a merge request to the `staging` branch

File: README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+1. `git push origin <your-branch>`
+2. create a merge request to the `stage branch`  
+
+Once the merge request is accepted, a git action is triggered to upload your changes to hubspot staging account  
+
+**TODO** put link of staging account
+
+### Go live
+
+Once the changes has been accepted create a merge request to `master branch`
```
Comment (ewansheldon) [inline README.md:70]: `master` branch?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+`npm install`
+
+Connect your local environment to your [developer sandbox](#developer-sandbox) :
+
+1. `npx hs init`
+2. On the prompt press enter, it opens a browser new page
+3. In the page, choose your developer sandbox account and click on **Continue with this account**
+4. copy the personal access key and paste it in your terminal
+5. enter a name that references your sandbox account : this name will be only visible by you
+6. check the file hubspot.config.yml has been created
```
Comment (ewansheldon) [inline README.md:24]: maybe put `hubspot.config.yml` in code block (but i'm not sure about that - that's just my preference)

**Review by ewansheldon — CHANGES_REQUESTED** [review]
generally good job man - the instructions were really easy to follow as i set up my sandbox

**Review by Morinslash — COMMENTED** [review]
I think it would be good to mention also that files removed from the repo are not removed from the HubSpot Accounts and they have to be removed manually inside porta

Adding links to documentation also will be good (CLI commands etc)

File: README.md (hunk 3 above)
Comment (nicolasbelfis) [reply README.md:24]: not sure either, if the file is created it is done by the hs process. But maybe we can later one point people to the doc that explains that file ?

**Review by ewansheldon — APPROVED** [review]
looks good to me now

**Review by Morinslash — APPROVED** [review]

**Pull request merged**